content/docs/access_rights/organization_account.md

```diff
@@ -52,15 +49,15 @@ Then, under the **Organizations** [tab](https://console.apify.com/account#/myorg
 
 Next, set the organization's name and click **Convert**.
 
-And that's it! Your personal account becomes the organization and you will be logged out automatically. You can now log into your new personal account with the same credentials as you are currently logged in with. This applies to both **password** and **OAuth** methods.
+And that's it! Your personal account becomes the organization, and you will be logged out automatically. You can now log into your new personal account with the same credentials as you are currently logged in with. This applies to both **password** and **OAuth** methods.
```

Comment:
No need for a comma here
```suggestion
And that's it! Your personal account becomes the organization and you will be logged out automatically. You can now log into your new personal account with the same credentials as you are currently logged in with. This applies to both **password** and **OAuth** methods.
```

Reply:
@m-murasovs I don't wanna be annoying, but isn't it exactly the case frmo the article, lol?

> Here’s a tip: Remember, when you’re joining two independent clauses, you need both a comma and a conjunction. If you use a comma without a conjunction, you’ll end up with a [comma splice](https://www.grammarly.com/blog/comma-splice/).

> Let’s look at another example.
It’s cold outside, and I can’t find my coat.

Reply:
Yeah, fair point 👍